liquidation-monitor/src/agent.js

```diff
@@ -111,6 +129,12 @@ function provideInitialize(data) {
     data.newAccounts = []; // New account from transaction events
     data.totalNewAccounts = 0;
 
+    // Calculate the next report time.
+    const latestBlockTimestamp = (await data.provider.getBlock('latest')).timestamp.toString();
+    //   Now minus seconds elapsed since midnight plus 1 day.
+    data.nextAlertTime = latestBlockTimestamp - (latestBlockTimestamp % 86400) + 86400;
```

Comment:
Oddly enough, this order of operations works, but I think we just got lucky:
```suggestion
    const latestBlockTimestamp = (await data.provider.getBlock('latest')).timestamp;
    //   Now minus seconds elapsed since midnight plus 1 day.
    data.nextAlertTime = latestBlockTimestamp - (latestBlockTimestamp % 86400) + 86400;
```

For example, the results are different if we move the operations into any other order:
```javascript
> const latestBlockTimestamp = 100;
> latestBlockTimestamp - (latestBlockTimestamp % 7) + 7
105
> latestBlockTimestamp.toString() - (latestBlockTimestamp.toString() % 7) + 7  // current ordering
105
> latestBlockTimestamp.toString() + 7 - (latestBlockTimestamp.toString() % 7)  // move the number forward
1005
> 7 + latestBlockTimestamp.toString() - (latestBlockTimestamp.toString() % 7)  // move the number to the front
7098
> 7 - (latestBlockTimestamp.toString() % 7) + latestBlockTimestamp.toString()  // swap the last two terms
'5100'
> - (latestBlockTimestamp.toString() % 7) + 7 + latestBlockTimestamp.toString()  // swap the first two terms
'5100'
> - (latestBlockTimestamp.toString() % 7) + latestBlockTimestamp.toString() + 7  // swap last two terms
'-21007'
```

Reply:
For some reason I thinking that the output was in ethers bigNumber and I wanted to get away from that. Although the minus does cast it back to number.  Good catch that cast to string is not needed here.